src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,15 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
+                statusToUser.add(new StatusMessage("Email sent to "
+                        + addressReceiverListString, StatusMessageColor.SUCCESS));
+            } else {
+                statusToAdmin = "Email sent to " + groupReceiverListFileKey;
+                statusToUser.add(new StatusMessage("Email sent to " + groupReceiverListFileKey, StatusMessageColor.SUCCESS));
+            }
```

Comment:
@xpdavid is going to look after the admin side of things. May be he can weigh in here?